packages/react-app/pages/_app.tsx

```diff
@@ -1,12 +1,52 @@
+import {
+  RainbowKitProvider,
+  connectorsForWallets,
+} from "@rainbow-me/rainbowkit";
+import "@rainbow-me/rainbowkit/styles.css";
+import { injectedWallet } from "@rainbow-me/rainbowkit/wallets";
 import type { AppProps } from "next/app";
+import { WagmiProvider, createConfig, http } from "wagmi";
+import { celo, celoAlfajores } from "wagmi/chains";
 import Layout from "../components/Layout";
 import "../styles/globals.css";
 
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const connectors = connectorsForWallets(
+  [
+    {
+      groupName: "Recommended",
+      wallets: [injectedWallet],
+    },
+  ],
+  {
+    appName: "Celo Composer",
+    projectId: "044601f65212332475a09bc14ceb3c34",
+  }
+);
+
+export const wagmiConfig = createConfig({
+  connectors,
+  chains: [celo, celoAlfajores],
+  transports: {
+    [celo.id]: http(),
```

Comment:
How about only allowing people to connect to Testnet only? Since the Mint NFT button does not work for Mainnet.